workers/unity/Packages/com.improbable.gdk.core/Collections/BlittableList.cs

```diff
@@ -0,0 +1,91 @@
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Improbable.Gdk.Core.Collections
+{
+    public struct BlittableList<TValue> : IList<TValue>
+    {
+        private readonly uint handle;
+
+        internal List<TValue> Internal => CollectionProvider<List<TValue>, TValue>.Get(handle);
+
+        internal BlittableList(uint handle)
+        {
+            this.handle = handle;
+        }
+
+        public static implicit operator List<TValue>(BlittableList<TValue> bl)
+        {
+            return bl.Internal;
+        }
+
+        #region IList<T>
```

Comment:
I'm still very against this unless there is some reason we need it. We want it to be a drop in replacement for List not a new IList.